Detectors/TPC/reconstruction/macro/convertTracks.C

```diff
@@ -8,19 +8,6 @@
 // granted to it by virtue of its status as an Intergovernmental Organization
 // or submit itself to any jurisdiction.
 
-#include <vector>
```

Comment:
this is likely not the solution to go. Instead of taking away those headers, they should be protected by 
```
#if (!defined(__CINT__) && !defined(__CLING__)) || defined(__MAKECINT__)
#endif
```